### ISSUE TITLE
Mac client -- error installing kext during setup

### ISSUE THREAD

**Issue by bennettp123** [keybase/client#8006]
### System details

OS Version: macOS High Sierra Version 10.13 Beta (17A315i)
Error message: "Keybase Error: KextManager failed to load with status -603946981"

This error is triggered by OS error "System Extension Blocked" (full message attached)
<img width="532" alt="screen shot 2017-08-04 at 7 39 42 am" src="https://user-images.githubusercontent.com/1610227/28948418-26eae726-78e8-11e7-9acb-beeeaa087950.png">

This is new behaviour in macOS 10.13: kexts are now blocked by default. More info: https://developer.apple.com/library/content/technotes/tn2459/_index.html

Workaround: System Preferences &rarr; Security & Privacy &rarr; Allow, then quit and relaunch the Keybase app:
<img width="780" alt="screen shot 2017-08-04 at 7 41 55 am" src="https://user-images.githubusercontent.com/1610227/28948494-99f617cc-78e8-11e7-9258-d03ca903602a.png">

(This is probably a duplicate of #7725, but the version numbers are different).

### Keybase error log

Installer: 1.1.57
Privileged Helper: Installed, Bundle Version: 1.0.29, Version: 1.0.29
Updater: Installed, PID: 21407
Service: Installed, Version: 1.0.26-20170724171020+b095a794e, PID: 21416
Fuse: Install, Install Error: KextManager failed to load with status: -603946981, Not Installed, Bundle Version: 3.6.0
Mount directory: Unknown, 
KBFS: Unknown, 
CLI: Unknown, 
KBNM: Unknown, 

Log:
Version: 1.1.57
Installables: (
    "Privileged Helper",
    Updater,
    Service,
    Fuse,
    "Mount directory",
    KBFS,
    CLI,
    KBNM
)
Install: Privileged Helper
Removing /Library/PrivilegedHelperTools/keybase.Helper
Installing helper tool via SMJobBless
Helper tool installed
Connecting to keybase.Helper (privileged=1)
Connected
Helper version: {
    build = 2;
    version = "1.0.29";
}
Install: Updater
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=updater --timeout=30s
Task EOF
Completed
Task (err): 2017-08-04T07:31:01.622339+08:00 ▶ [DEBU keybase globals.go:394] 001 Created Identify2Cache, max age: 5m0s
2017-08-04T07:31:01.622582+08:00 ▶ [DEBU keybase globals.go:397] 002 Created LinkCache, max size: 65536, clean dur: 1m0s
2017-08-04T07:31:01.622743+08:00 ▶ [DEBU keybase globals.go:399] 003 Created CardCache, max age: 5m0s
2017-08-04T07:31:01.622796+08:00 ▶ [DEBU keybase globals.go:401] 004 made a new full self cache
2017-08-04T07:31:01.622809+08:00 ▶ [DEBU keybase globals.go:403] 005 made a new cached UPAK loader (timeout=10m0s)
2017-08-04T07:31:01.639201+08:00 ▶ [DEBU keybase ui.go:580] 006 Can't set GPG_TTY; discover failed
2017-08-04T07:31:01.639286+08:00 ▶ [DEBU keybase globals.go:377] 007 Keybase CLI 1.0.26-20170724171020+b095a794e
2017-08-04T07:31:01.639315+08:00 ▶ [DEBU keybase globals.go:377] 008 - Built with go1.8.3
2017-08-04T07:31:01.639321+08:00 ▶ [DEBU keybase globals.go:377] 009 - Visit https://keybase.io for more details
2017-08-04T07:31:01.639330+08:00 ▶ [DEBU keybase main.go:152] 00a + configureProcesses
2017-08-04T07:31:01.639358+08:00 ▶ [DEBU keybase main.go:256] 00b + configureLogging
2017-08-04T07:31:01.639369+08:00 ▶ [DEBU keybase main.go:265] 00c Disabling log forwarding
2017-08-04T07:31:01.639376+08:00 ▶ [DEBU keybase main.go:258] 00d - configureLogging
2017-08-04T07:31:01.639382+08:00 ▶ [DEBU keybase main.go:154] 00e - configureProcesses -> <nil>
2017-08-04T07:31:01.639559+08:00 ▶ [DEBU keybase install_darwin.go:404] 00f Installing components: [updater]
2017-08-04T07:31:01.639701+08:00 ▶ [DEBU keybase install_darwin.go:998] 010 Using updater path: /Applications/Keybase.app/Contents/SharedSupport/bin/updater
2017-08-04T07:31:01.649471+08:00 ▶ [DEBU keybase launchd.go:141] 011 Output (launchctl remove): 
2017-08-04T07:31:01.649583+08:00 ▶ [DEBU keybase launchd.go:146] 012 Nothing to stop (keybase.updater)
2017-08-04T07:31:01.649605+08:00 ▶ [DEBU keybase install_darwin.go:1008] 013 Installing updater service (keybase.updater, timeout=30s)
2017-08-04T07:31:01.649834+08:00 ▶ [INFO keybase launchd.go:270] 014 Saving /Users/bennett/Library/LaunchAgents/keybase.updater.plist
2017-08-04T07:31:01.649945+08:00 ▶ [DEBU keybase util.go:181] 015 + Writing to /Users/bennett/Library/LaunchAgents/keybase.updater.plist
2017-08-04T07:31:01.650227+08:00 ▶ [DEBU keybase util.go:183] 016 | Temporary file generated: /Users/bennett/Library/LaunchAgents/keybase.updater.plist.RT5LZVTDJ6DOITRFBMZXTWQNXO6ZYFJ2
2017-08-04T07:31:01.650606+08:00 ▶ [DEBU keybase util.go:206] 017 - Wrote to /Users/bennett/Library/LaunchAgents/keybase.updater.plist -> ok
2017-08-04T07:31:01.650692+08:00 ▶ [INFO keybase launchd.go:95] 018 Starting keybase.updater
2017-08-04T07:31:01.659863+08:00 ▶ [DEBU keybase launchd.go:98] 019 Output (launchctl load): 
2017-08-04T07:31:01.659919+08:00 ▶ [INFO keybase launchd.go:176] 01a Waiting for keybase.updater to be loaded...
2017-08-04T07:31:02.169279+08:00 ▶ [DEBU keybase launchd.go:111] 01b Service status: &launchd.ServiceStatus{label:"keybase.updater", pid:"21407", lastExitStatus:""}
2017-08-04T07:31:02.689087+08:00 ▶ [DEBU keybase out_of_date.go:14] 01c + PrintOutOfDateWarnings
2017-08-04T07:31:02.689280+08:00 ▶ [DEBU keybase util.go:423] 01d + GetSocket
2017-08-04T07:31:02.689441+08:00 ▶ [DEBU keybase util.go:424] 01e - GetSocket -> ok
2017-08-04T07:31:02.689497+08:00 ▶ [DEBU keybase socket.go:73] 01f | empty socket wrapper; need a new one
2017-08-04T07:31:02.689659+08:00 ▶ [DEBU keybase socket_nix.go:96] 020 Dialing unix:/Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:02.691022+08:00 ▶ [DEBU keybase socket_nix.go:96] 021 Dialing unix:/Users/bennett/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:02.691634+08:00 ▶ [DEBU keybase socket.go:88] 022 | DialSocket -> ERROR: There were multiple errors: dial unix /Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock: connect: no such file or directory; dial unix /Users/bennett/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2017-08-04T07:31:02.691684+08:00 ▶ [DEBU keybase out_of_date.go:19] 023 Ignoring error in printOutOfDateWarnings: There were multiple errors: dial unix /Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock: connect: no such file or directory; dial unix /Users/bennett/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2017-08-04T07:31:02.691821+08:00 ▶ [DEBU keybase out_of_date.go:20] 024 - PrintOutOfDateWarnings
2017-08-04T07:31:02.796308+08:00 ▶ [DEBU keybase globals.go:481] 025 Calling shutdown first time through
2017-08-04T07:31:02.796577+08:00 ▶ [DEBU keybase login_state.go:1095] 026 + Account "LoginState - Shutdown"
2017-08-04T07:31:02.796728+08:00 ▶ [DEBU keybase login_state.go:942] 027 + acctHandle sending: acctReq LoginState - Shutdown [viDQ9GLt]
2017-08-04T07:31:02.796820+08:00 ▶ [DEBU keybase login_state.go:945] 028 * acctHandle sent: acctReq LoginState - Shutdown [viDQ9GLt] (164µs)
2017-08-04T07:31:02.796900+08:00 ▶ [DEBU keybase login_state.go:997] 029 * LoginState running account request: acctReq LoginState - Shutdown [viDQ9GLt]
2017-08-04T07:31:02.797137+08:00 ▶ [DEBU keybase login_state.go:1000] 02a * LoginState account request complete: acctReq LoginState - Shutdown [viDQ9GLt]
2017-08-04T07:31:02.797249+08:00 ▶ [DEBU keybase login_state.go:980] 02c - LoginState: shutdown chan closed, exiting requests loop
2017-08-04T07:31:02.797297+08:00 ▶ [DEBU keybase login_state.go:981] 02d - LoginState: Leaving request loop
2017-08-04T07:31:02.797191+08:00 ▶ [DEBU keybase login_state.go:960] 02b - acctHandle done: acctReq LoginState - Shutdown [viDQ9GLt] (689µs)
2017-08-04T07:31:02.797427+08:00 ▶ [DEBU keybase login_state.go:1097] 02e - Account "LoginState - Shutdown"
2017-08-04T07:31:02.797553+08:00 ▶ [DEBU keybase globals.go:540] 02f exiting shutdown code=0; err=<nil>

Task (out): {
  "componentResults": [
    {
      "name": "updater",
      "status": {
        "code": 0,
        "name": "OK",
        "desc": "OK",
        "fields": null
      }
    }
  ],
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  },
  "fatal": false
}

Checking launchd status for updater
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file launchd status --format=json --timeout=30s updater
Task EOF
Completed
Task (err): 2017-08-04T07:31:02.895102+08:00 ▶ [DEBU keybase globals.go:394] 001 Created Identify2Cache, max age: 5m0s
2017-08-04T07:31:02.895203+08:00 ▶ [DEBU keybase globals.go:397] 002 Created LinkCache, max size: 65536, clean dur: 1m0s
2017-08-04T07:31:02.895238+08:00 ▶ [DEBU keybase globals.go:399] 003 Created CardCache, max age: 5m0s
2017-08-04T07:31:02.895259+08:00 ▶ [DEBU keybase globals.go:401] 004 made a new full self cache
2017-08-04T07:31:02.895275+08:00 ▶ [DEBU keybase globals.go:403] 005 made a new cached UPAK loader (timeout=10m0s)
2017-08-04T07:31:02.899236+08:00 ▶ [DEBU keybase ui.go:580] 006 Can't set GPG_TTY; discover failed
2017-08-04T07:31:02.899281+08:00 ▶ [DEBU keybase globals.go:377] 007 Keybase CLI 1.0.26-20170724171020+b095a794e
2017-08-04T07:31:02.899303+08:00 ▶ [DEBU keybase globals.go:377] 008 - Built with go1.8.3
2017-08-04T07:31:02.899313+08:00 ▶ [DEBU keybase globals.go:377] 009 - Visit https://keybase.io for more details
2017-08-04T07:31:02.899329+08:00 ▶ [DEBU keybase main.go:152] 00a + configureProcesses
2017-08-04T07:31:02.899352+08:00 ▶ [DEBU keybase main.go:256] 00b + configureLogging
2017-08-04T07:31:02.899382+08:00 ▶ [DEBU keybase main.go:265] 00c Disabling log forwarding
2017-08-04T07:31:02.899398+08:00 ▶ [DEBU keybase main.go:258] 00d - configureLogging
2017-08-04T07:31:02.899411+08:00 ▶ [DEBU keybase main.go:154] 00e - configureProcesses -> <nil>
2017-08-04T07:31:03.414109+08:00 ▶ [DEBU keybase out_of_date.go:14] 00f + PrintOutOfDateWarnings
2017-08-04T07:31:03.414222+08:00 ▶ [DEBU keybase util.go:423] 010 + GetSocket
2017-08-04T07:31:03.414249+08:00 ▶ [DEBU keybase util.go:424] 011 - GetSocket -> ok
2017-08-04T07:31:03.414267+08:00 ▶ [DEBU keybase socket.go:73] 012 | empty socket wrapper; need a new one
2017-08-04T07:31:03.414294+08:00 ▶ [DEBU keybase socket_nix.go:96] 013 Dialing unix:/Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:03.414657+08:00 ▶ [DEBU keybase socket_nix.go:96] 014 Dialing unix:/Users/bennett/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:03.414844+08:00 ▶ [DEBU keybase socket.go:88] 015 | DialSocket -> ERROR: There were multiple errors: dial unix /Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock: connect: no such file or directory; dial unix /Users/bennett/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2017-08-04T07:31:03.414871+08:00 ▶ [DEBU keybase out_of_date.go:19] 016 Ignoring error in printOutOfDateWarnings: There were multiple errors: dial unix /Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock: connect: no such file or directory; dial unix /Users/bennett/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2017-08-04T07:31:03.414899+08:00 ▶ [DEBU keybase out_of_date.go:20] 017 - PrintOutOfDateWarnings
2017-08-04T07:31:03.518662+08:00 ▶ [DEBU keybase globals.go:481] 018 Calling shutdown first time through
2017-08-04T07:31:03.518796+08:00 ▶ [DEBU keybase login_state.go:1095] 019 + Account "LoginState - Shutdown"
2017-08-04T07:31:03.518877+08:00 ▶ [DEBU keybase login_state.go:942] 01a + acctHandle sending: acctReq LoginState - Shutdown [z1SfHfha]
2017-08-04T07:31:03.518947+08:00 ▶ [DEBU keybase login_state.go:945] 01b * acctHandle sent: acctReq LoginState - Shutdown [z1SfHfha] (116µs)
2017-08-04T07:31:03.519032+08:00 ▶ [DEBU keybase login_state.go:997] 01c * LoginState running account request: acctReq LoginState - Shutdown [z1SfHfha]
2017-08-04T07:31:03.519180+08:00 ▶ [DEBU keybase login_state.go:1000] 01d * LoginState account request complete: acctReq LoginState - Shutdown [z1SfHfha]
2017-08-04T07:31:03.519227+08:00 ▶ [DEBU keybase login_state.go:960] 01e - acctHandle done: acctReq LoginState - Shutdown [z1SfHfha] (407µs)
2017-08-04T07:31:03.519314+08:00 ▶ [DEBU keybase login_state.go:1097] 020 - Account "LoginState - Shutdown"
2017-08-04T07:31:03.519239+08:00 ▶ [DEBU keybase login_state.go:980] 01f - LoginState: shutdown chan closed, exiting requests loop
2017-08-04T07:31:03.519483+08:00 ▶ [DEBU keybase login_state.go:981] 022 - LoginState: Leaving request loop
2017-08-04T07:31:03.519419+08:00 ▶ [DEBU keybase globals.go:540] 021 exiting shutdown code=0; err=<nil>

Task (out): {
  "version": "",
  "label": "keybase.updater",
  "pid": "21407",
  "lastExitStatus": "",
  "bundleVersion": "",
  "installStatus": 4,
  "installAction": 1,
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  }
}

Install: Service
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service --timeout=30s
Task EOF
Completed
Task (err): 2017-08-04T07:31:03.619776+08:00 ▶ [DEBU keybase globals.go:394] 001 Created Identify2Cache, max age: 5m0s
2017-08-04T07:31:03.619864+08:00 ▶ [DEBU keybase globals.go:397] 002 Created LinkCache, max size: 65536, clean dur: 1m0s
2017-08-04T07:31:03.619917+08:00 ▶ [DEBU keybase globals.go:399] 003 Created CardCache, max age: 5m0s
2017-08-04T07:31:03.619933+08:00 ▶ [DEBU keybase globals.go:401] 004 made a new full self cache
2017-08-04T07:31:03.619965+08:00 ▶ [DEBU keybase globals.go:403] 005 made a new cached UPAK loader (timeout=10m0s)
2017-08-04T07:31:03.623795+08:00 ▶ [DEBU keybase ui.go:580] 006 Can't set GPG_TTY; discover failed
2017-08-04T07:31:03.623851+08:00 ▶ [DEBU keybase globals.go:377] 007 Keybase CLI 1.0.26-20170724171020+b095a794e
2017-08-04T07:31:03.623873+08:00 ▶ [DEBU keybase globals.go:377] 008 - Built with go1.8.3
2017-08-04T07:31:03.623884+08:00 ▶ [DEBU keybase globals.go:377] 009 - Visit https://keybase.io for more details
2017-08-04T07:31:03.623900+08:00 ▶ [DEBU keybase main.go:152] 00a + configureProcesses
2017-08-04T07:31:03.623923+08:00 ▶ [DEBU keybase main.go:256] 00b + configureLogging
2017-08-04T07:31:03.623944+08:00 ▶ [DEBU keybase main.go:265] 00c Disabling log forwarding
2017-08-04T07:31:03.623961+08:00 ▶ [DEBU keybase main.go:258] 00d - configureLogging
2017-08-04T07:31:03.623976+08:00 ▶ [DEBU keybase main.go:154] 00e - configureProcesses -> <nil>
2017-08-04T07:31:03.624117+08:00 ▶ [DEBU keybase install_darwin.go:404] 00f Installing components: [service]
2017-08-04T07:31:03.624224+08:00 ▶ [DEBU keybase install_darwin.go:527] 010 Using binPath: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase
2017-08-04T07:31:03.629430+08:00 ▶ [DEBU keybase launchd.go:141] 011 Output (launchctl remove): 
2017-08-04T07:31:03.629484+08:00 ▶ [DEBU keybase launchd.go:146] 012 Nothing to stop (keybase.service)
2017-08-04T07:31:03.633720+08:00 ▶ [DEBU keybase launchd.go:141] 013 Output (launchctl remove): 
2017-08-04T07:31:03.633769+08:00 ▶ [DEBU keybase launchd.go:146] 014 Nothing to stop (homebrew.mxcl.keybase)
2017-08-04T07:31:03.633793+08:00 ▶ [DEBU keybase install_darwin.go:536] 015 Installing service (keybase.service, timeout=30s)
2017-08-04T07:31:03.633909+08:00 ▶ [INFO keybase launchd.go:270] 016 Saving /Users/bennett/Library/LaunchAgents/keybase.service.plist
2017-08-04T07:31:03.633943+08:00 ▶ [DEBU keybase util.go:181] 017 + Writing to /Users/bennett/Library/LaunchAgents/keybase.service.plist
2017-08-04T07:31:03.634102+08:00 ▶ [DEBU keybase util.go:183] 018 | Temporary file generated: /Users/bennett/Library/LaunchAgents/keybase.service.plist.SWFAHOY4BGE2LAUS32I3RWPRUOOCCPTF
2017-08-04T07:31:03.634401+08:00 ▶ [DEBU keybase util.go:206] 019 - Wrote to /Users/bennett/Library/LaunchAgents/keybase.service.plist -> ok
2017-08-04T07:31:03.634459+08:00 ▶ [INFO keybase launchd.go:95] 01a Starting keybase.service
2017-08-04T07:31:03.641298+08:00 ▶ [DEBU keybase launchd.go:98] 01b Output (launchctl load): 
2017-08-04T07:31:03.641364+08:00 ▶ [INFO keybase launchd.go:176] 01c Waiting for keybase.service to be loaded...
2017-08-04T07:31:04.153885+08:00 ▶ [DEBU keybase launchd.go:111] 01d Service status: &launchd.ServiceStatus{label:"keybase.service", pid:"21416", lastExitStatus:""}
2017-08-04T07:31:04.670363+08:00 ▶ [DEBU keybase service_info.go:92] 01e Looking for service info file (timeout=30s)
2017-08-04T07:31:05.075876+08:00 ▶ [DEBU keybase service_info.go:104] 01f Found service info: libkb.ServiceInfo{Version:"1.0.26-20170724171020+b095a794e", Label:"keybase.service", Pid:21416}
2017-08-04T07:31:05.076188+08:00 ▶ [DEBU keybase out_of_date.go:14] 020 + PrintOutOfDateWarnings
2017-08-04T07:31:05.076236+08:00 ▶ [DEBU keybase util.go:423] 021 + GetSocket
2017-08-04T07:31:05.076284+08:00 ▶ [DEBU keybase util.go:424] 022 - GetSocket -> ok
2017-08-04T07:31:05.076321+08:00 ▶ [DEBU keybase socket.go:73] 023 | empty socket wrapper; need a new one
2017-08-04T07:31:05.076362+08:00 ▶ [DEBU keybase socket_nix.go:96] 024 Dialing unix:/Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:05.076586+08:00 ▶ [DEBU keybase socket.go:88] 025 | DialSocket -> ok
2017-08-04T07:31:05.084919+08:00 ▶ [DEBU keybase out_of_date.go:28] 026 Got OutOfDateInfo: keybase1.OutOfDateInfo{UpgradeTo:"", UpgradeURI:"", CustomMessage:"", CriticalClockSkew:0}
2017-08-04T07:31:05.084991+08:00 ▶ [DEBU keybase out_of_date.go:44] 027 - PrintOutOfDateWarnings
2017-08-04T07:31:05.186569+08:00 ▶ [DEBU keybase globals.go:481] 028 Calling shutdown first time through
2017-08-04T07:31:05.186691+08:00 ▶ [DEBU keybase login_state.go:1095] 029 + Account "LoginState - Shutdown"
2017-08-04T07:31:05.186772+08:00 ▶ [DEBU keybase login_state.go:942] 02a + acctHandle sending: acctReq LoginState - Shutdown [vn43YtKd]
2017-08-04T07:31:05.186862+08:00 ▶ [DEBU keybase login_state.go:945] 02b * acctHandle sent: acctReq LoginState - Shutdown [vn43YtKd] (147µs)
2017-08-04T07:31:05.186949+08:00 ▶ [DEBU keybase login_state.go:997] 02c * LoginState running account request: acctReq LoginState - Shutdown [vn43YtKd]
2017-08-04T07:31:05.187051+08:00 ▶ [DEBU keybase login_state.go:1000] 02d * LoginState account request complete: acctReq LoginState - Shutdown [vn43YtKd]
2017-08-04T07:31:05.187108+08:00 ▶ [DEBU keybase login_state.go:960] 02e - acctHandle done: acctReq LoginState - Shutdown [vn43YtKd] (664µs)
2017-08-04T07:31:05.187469+08:00 ▶ [DEBU keybase login_state.go:1097] 030 - Account "LoginState - Shutdown"
2017-08-04T07:31:05.187150+08:00 ▶ [DEBU keybase login_state.go:980] 02f - LoginState: shutdown chan closed, exiting requests loop
2017-08-04T07:31:05.187561+08:00 ▶ [DEBU keybase globals.go:540] 031 exiting shutdown code=0; err=<nil>

Task (out): {
  "componentResults": [
    {
      "name": "service",
      "status": {
        "code": 0,
        "name": "OK",
        "desc": "OK",
        "fields": null
      }
    }
  ],
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  },
  "fatal": false
}

Checking launchd status for service
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file launchd status --format=json --timeout=30s service
Task EOF
Completed
Task (err): 2017-08-04T07:31:05.280589+08:00 ▶ [DEBU keybase globals.go:394] 001 Created Identify2Cache, max age: 5m0s
2017-08-04T07:31:05.280681+08:00 ▶ [DEBU keybase globals.go:397] 002 Created LinkCache, max size: 65536, clean dur: 1m0s
2017-08-04T07:31:05.280715+08:00 ▶ [DEBU keybase globals.go:399] 003 Created CardCache, max age: 5m0s
2017-08-04T07:31:05.280737+08:00 ▶ [DEBU keybase globals.go:401] 004 made a new full self cache
2017-08-04T07:31:05.280755+08:00 ▶ [DEBU keybase globals.go:403] 005 made a new cached UPAK loader (timeout=10m0s)
2017-08-04T07:31:05.284634+08:00 ▶ [DEBU keybase ui.go:580] 006 Can't set GPG_TTY; discover failed
2017-08-04T07:31:05.284680+08:00 ▶ [DEBU keybase globals.go:377] 007 Keybase CLI 1.0.26-20170724171020+b095a794e
2017-08-04T07:31:05.284695+08:00 ▶ [DEBU keybase globals.go:377] 008 - Built with go1.8.3
2017-08-04T07:31:05.284703+08:00 ▶ [DEBU keybase globals.go:377] 009 - Visit https://keybase.io for more details
2017-08-04T07:31:05.284713+08:00 ▶ [DEBU keybase main.go:152] 00a + configureProcesses
2017-08-04T07:31:05.284732+08:00 ▶ [DEBU keybase main.go:256] 00b + configureLogging
2017-08-04T07:31:05.284748+08:00 ▶ [DEBU keybase main.go:265] 00c Disabling log forwarding
2017-08-04T07:31:05.284771+08:00 ▶ [DEBU keybase main.go:258] 00d - configureLogging
2017-08-04T07:31:05.284790+08:00 ▶ [DEBU keybase main.go:154] 00e - configureProcesses -> <nil>
2017-08-04T07:31:05.800317+08:00 ▶ [DEBU keybase service_info.go:92] 00f Looking for service info file (timeout=30s)
2017-08-04T07:31:06.205307+08:00 ▶ [DEBU keybase service_info.go:104] 010 Found service info: libkb.ServiceInfo{Version:"1.0.26-20170724171020+b095a794e", Label:"keybase.service", Pid:21416}
2017-08-04T07:31:06.205848+08:00 ▶ [DEBU keybase out_of_date.go:14] 011 + PrintOutOfDateWarnings
2017-08-04T07:31:06.205909+08:00 ▶ [DEBU keybase util.go:423] 012 + GetSocket
2017-08-04T07:31:06.205964+08:00 ▶ [DEBU keybase util.go:424] 013 - GetSocket -> ok
2017-08-04T07:31:06.206001+08:00 ▶ [DEBU keybase socket.go:73] 014 | empty socket wrapper; need a new one
2017-08-04T07:31:06.206048+08:00 ▶ [DEBU keybase socket_nix.go:96] 015 Dialing unix:/Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:06.206261+08:00 ▶ [DEBU keybase socket.go:88] 016 | DialSocket -> ok
2017-08-04T07:31:06.208631+08:00 ▶ [DEBU keybase out_of_date.go:28] 017 Got OutOfDateInfo: keybase1.OutOfDateInfo{UpgradeTo:"", UpgradeURI:"", CustomMessage:"", CriticalClockSkew:0}
2017-08-04T07:31:06.208713+08:00 ▶ [DEBU keybase out_of_date.go:44] 018 - PrintOutOfDateWarnings
2017-08-04T07:31:06.312703+08:00 ▶ [DEBU keybase globals.go:481] 019 Calling shutdown first time through
2017-08-04T07:31:06.312835+08:00 ▶ [DEBU keybase login_state.go:1095] 01a + Account "LoginState - Shutdown"
2017-08-04T07:31:06.312886+08:00 ▶ [DEBU keybase login_state.go:942] 01b + acctHandle sending: acctReq LoginState - Shutdown [cpsoOprL]
2017-08-04T07:31:06.312934+08:00 ▶ [DEBU keybase login_state.go:945] 01c * acctHandle sent: acctReq LoginState - Shutdown [cpsoOprL] (96µs)
2017-08-04T07:31:06.313020+08:00 ▶ [DEBU keybase login_state.go:997] 01d * LoginState running account request: acctReq LoginState - Shutdown [cpsoOprL]
2017-08-04T07:31:06.313073+08:00 ▶ [DEBU keybase login_state.go:1000] 01e * LoginState account request complete: acctReq LoginState - Shutdown [cpsoOprL]
2017-08-04T07:31:06.313127+08:00 ▶ [DEBU keybase login_state.go:980] 01f - LoginState: shutdown chan closed, exiting requests loop
2017-08-04T07:31:06.313132+08:00 ▶ [DEBU keybase login_state.go:960] 020 - acctHandle done: acctReq LoginState - Shutdown [cpsoOprL] (338µs)
2017-08-04T07:31:06.313254+08:00 ▶ [DEBU keybase login_state.go:1097] 022 - Account "LoginState - Shutdown"
2017-08-04T07:31:06.313171+08:00 ▶ [DEBU keybase login_state.go:981] 021 - LoginState: Leaving request loop
2017-08-04T07:31:06.313366+08:00 ▶ [DEBU keybase globals.go:540] 023 exiting shutdown code=0; err=<nil>

Task (out): {
  "version": "1.0.26-20170724171020+b095a794e",
  "label": "keybase.service",
  "pid": "21416",
  "lastExitStatus": "",
  "bundleVersion": "1.0.26-20170724171020+b095a794e",
  "installStatus": 4,
  "installAction": 1,
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  }
}

Install: Fuse
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file fuse status --bundle-version=3.6.0
Task EOF
Completed
Task (err): 2017-08-04T07:31:06.443400+08:00 ▶ [DEBU keybase globals.go:394] 001 Created Identify2Cache, max age: 5m0s
2017-08-04T07:31:06.443482+08:00 ▶ [DEBU keybase globals.go:397] 002 Created LinkCache, max size: 65536, clean dur: 1m0s
2017-08-04T07:31:06.443512+08:00 ▶ [DEBU keybase globals.go:399] 003 Created CardCache, max age: 5m0s
2017-08-04T07:31:06.443531+08:00 ▶ [DEBU keybase globals.go:401] 004 made a new full self cache
2017-08-04T07:31:06.443546+08:00 ▶ [DEBU keybase globals.go:403] 005 made a new cached UPAK loader (timeout=10m0s)
2017-08-04T07:31:06.447936+08:00 ▶ [DEBU keybase ui.go:580] 006 Can't set GPG_TTY; discover failed
2017-08-04T07:31:06.447997+08:00 ▶ [DEBU keybase globals.go:377] 007 Keybase CLI 1.0.26-20170724171020+b095a794e
2017-08-04T07:31:06.448017+08:00 ▶ [DEBU keybase globals.go:377] 008 - Built with go1.8.3
2017-08-04T07:31:06.448031+08:00 ▶ [DEBU keybase globals.go:377] 009 - Visit https://keybase.io for more details
2017-08-04T07:31:06.448049+08:00 ▶ [DEBU keybase main.go:152] 00a + configureProcesses
2017-08-04T07:31:06.448074+08:00 ▶ [DEBU keybase main.go:256] 00b + configureLogging
2017-08-04T07:31:06.448088+08:00 ▶ [DEBU keybase main.go:265] 00c Disabling log forwarding
2017-08-04T07:31:06.448096+08:00 ▶ [DEBU keybase main.go:258] 00d - configureLogging
2017-08-04T07:31:06.448103+08:00 ▶ [DEBU keybase main.go:154] 00e - configureProcesses -> <nil>
2017-08-04T07:31:06.448337+08:00 ▶ [DEBU keybase out_of_date.go:14] 00f + PrintOutOfDateWarnings
2017-08-04T07:31:06.448359+08:00 ▶ [DEBU keybase util.go:423] 010 + GetSocket
2017-08-04T07:31:06.448378+08:00 ▶ [DEBU keybase util.go:424] 011 - GetSocket -> ok
2017-08-04T07:31:06.448392+08:00 ▶ [DEBU keybase socket.go:73] 012 | empty socket wrapper; need a new one
2017-08-04T07:31:06.448414+08:00 ▶ [DEBU keybase socket_nix.go:96] 013 Dialing unix:/Users/bennett/Library/Group Containers/keybase/Library/Caches/Keybase/keybased.sock
2017-08-04T07:31:06.448505+08:00 ▶ [DEBU keybase socket.go:88] 014 | DialSocket -> ok
2017-08-04T07:31:06.450630+08:00 ▶ [DEBU keybase out_of_date.go:28] 015 Got OutOfDateInfo: keybase1.OutOfDateInfo{UpgradeTo:"", UpgradeURI:"", CustomMessage:"", CriticalClockSkew:0}
2017-08-04T07:31:06.450687+08:00 ▶ [DEBU keybase out_of_date.go:44] 016 - PrintOutOfDateWarnings
2017-08-04T07:31:06.555467+08:00 ▶ [DEBU keybase globals.go:481] 017 Calling shutdown first time through
2017-08-04T07:31:06.555603+08:00 ▶ [DEBU keybase login_state.go:1095] 018 + Account "LoginState - Shutdown"
2017-08-04T07:31:06.555682+08:00 ▶ [DEBU keybase login_state.go:942] 019 + acctHandle sending: acctReq LoginState - Shutdown [p2YllkQY]
2017-08-04T07:31:06.555746+08:00 ▶ [DEBU keybase login_state.go:945] 01a * acctHandle sent: acctReq LoginState - Shutdown [p2YllkQY] (111µs)
2017-08-04T07:31:06.555856+08:00 ▶ [DEBU keybase login_state.go:997] 01b * LoginState running account request: acctReq LoginState - Shutdown [p2YllkQY]
2017-08-04T07:31:06.556001+08:00 ▶ [DEBU keybase login_state.go:1000] 01c * LoginState account request complete: acctReq LoginState - Shutdown [p2YllkQY]
2017-08-04T07:31:06.556058+08:00 ▶ [DEBU keybase login_state.go:960] 01d - acctHandle done: acctReq LoginState - Shutdown [p2YllkQY] (428µs)
2017-08-04T07:31:06.556221+08:00 ▶ [DEBU keybase login_state.go:1097] 01f - Account "LoginState - Shutdown"
2017-08-04T07:31:06.556337+08:00 ▶ [DEBU keybase globals.go:540] 020 exiting shutdown code=0; err=<nil>

Task (out): {
  "version": "",
  "bundleVersion": "3.6.0",
  "kextID": "",
  "path": "",
  "kextStarted": false,
  "installStatus": 2,
  "installAction": 4,
  "mountInfos": null,
  "status": {
    "code": 0,
    "name": "",
    "desc": "",
    "fields": null
  }
}

Helper: kextInstall({
    destination = "/Library/Filesystems/kbfuse.fs";
    kextID = "com.github.kbfuse.filesystems.kbfuse";
    kextPath = "/Library/Filesystems/kbfuse.fs/Contents/Extensions/10.10/kbfuse.kext";
    source = "/Applications/Keybase.app/Contents/Resources/KeybaseInstaller.app/Contents/Resources/kbfuse.bundle";
})
Install complete
Privileged Helper: Installed, Bundle Version: 1.0.29
Version: 1.0.29
Updater: Installed, PID: 21407
Service: Installed, Version: 1.0.26-20170724171020+b095a794e
PID: 21416
Fuse: Install Error: KextManager failed to load with status: -603946981
Not Installed, Bundle Version: 3.6.0
